### ISSUE TITLE
picked up the Ebony Blade at dragonsreach

### ISSUE THREAD

**Issue by SteveTownsend** [SteveTownsend/SmartHarvestSE#136]
nexus report, just as it says in the title

**Comment by SteveTownsend** [comment]
I cannot reproduce this without settings so permissive I would not use them
```
J:\GitHub\SmartHarvestSE\src\Looting\ScanGovernor.cpp(275): [VERBOSE] {20964} (2020-07-17 11:21:48 AM) Process REFR 0x0004a39d with base object Ebony Blade/0x0004a38f
J:\GitHub\SmartHarvestSE\src\Utilities\debugs.cpp(155): [VERBOSE] {20964} (2020-07-17 11:21:48 AM) 0x0004a38f 0x29(41) [Ebony Blade] - typename clutter
J:\GitHub\SmartHarvestSE\src\Utilities\debugs.cpp(147): [MESSAGE] {20964} (2020-07-17 11:21:48 AM) Worth(33841.00)  Weight(14.00)  V/W(2417.21)
J:\GitHub\SmartHarvestSE\src\FormHelpers\ExtraDataListHelper.cpp(46): [VERBOSE] {20964} (2020-07-17 11:21:48 AM) Quest Item confirmed in alias for quest 0x0004a37b, alias quest object true
J:\GitHub\SmartHarvestSE\src\Looting\TryLootREFR.cpp(121): [VERBOSE] {20964} (2020-07-17 11:21:48 AM) Quest Item 0x0004a38f
J:\GitHub\SmartHarvestSE\src\Looting\TryLootREFR.cpp(124): [VERBOSE] {20964} (2020-07-17 11:21:48 AM) glow quest object Ebony Blade/0x0004a38f
J:\GitHub\SmartHarvestSE\src\Looting\TryLootREFR.cpp(141): [VERBOSE] {20964} (2020-07-17 11:21:48 AM) Collectible Item 0x0004a38f
J:\GitHub\SmartHarvestSE\src\FormHelpers\IHasValueWeight.cpp(114): [VERBOSE] {20964} (2020-07-17 11:21:48 AM) Ebony Blade/0004a38f has value 33841 vs threshold 500.00: Valuable
J:\GitHub\SmartHarvestSE\src\Looting\TryLootREFR.cpp(155): [VERBOSE] {20964} (2020-07-17 11:21:48 AM) Valuable Item 0x0004a38f
J:\GitHub\SmartHarvestSE\src\Looting\TryLootREFR.cpp(640): [VERBOSE] {20964} (2020-07-17 11:21:48 AM) Player-owned true, looting belongings disallowed: Ebony Blade/0x0004a38f
```

**Comment by SteveTownsend** [comment]
Need to revisit this with latest logic. A lot has changed.

**Comment by SteveTownsend** [comment]
I still autoharvest this, because it's now in a pickup-always Collection (one of my test ones)
```
2020-08-05 09:00:48.448    trace  35524 Process REFR 0x0004a39d with base object Ebony Blade/0x0004a38f
2020-08-05 09:00:48.448    trace  35524 Collectible Item 0x0004a38f
2020-08-05 09:00:48.448    trace  35524 Quest Item confirmed in alias for quest 0x0004a37b, alias quest object true
2020-08-05 09:00:48.448    trace  35524 glow quest object Ebony Blade/0x0004a38f
2020-08-05 09:00:48.448    trace  35524 Ebony Blade/0004a38f has value 4800 vs threshold 500.00: Valuable
2020-08-05 09:00:48.449    trace  35524 Valuable Item 0x0004a38f
2020-08-05 09:00:48.449    trace  35524 Player-owned true, looting belongings disallowed: Ebony Blade/0x0004a38f
2020-08-05 09:00:48.449    trace  35524 Collectible REFR 0x0004a39d overrides Legality PlayerOwned for Ebony Blade/0x0004a38f
2020-08-05 09:00:48.449    trace  35524 Trigger glow for Ebony Blade/0x0004a39d
2020-08-05 09:00:48.449    trace  35524 check bound object Ebony Blade/0x0004a38f with type 41
2020-08-05 09:00:48.449    trace  35524 Player location is excluded as restricted population center for this item
2020-08-05 09:00:48.449    trace  35524 Lootable REFR to collectible 0x0004a38f, skip = false
2020-08-05 09:00:48.449    trace  35524 SmartHarvest Ebony Blade/0x0004a38f for REFR 0x0004a39d, collectible=true
```

**Comment by SteveTownsend** [comment]
The only thing for me that prevents this from getting looted as above is the 'nearby door' range restriction. I think the item is placed out of reach until the related quest enables it, but the range checking of SHSE does not know anything about 50 feet of intervening rock. Before I disable the door range restriction this REFR is always ruled out as too far away. As soon as I disable it, the item is picked up as part of a Collection. 
I think the logic has a bug though - this item is now correctly identified as a QUST object and glowed. I don't think Collections membership should override this.
```
020-08-05 09:00:48.448    trace  35524 Quest Item confirmed in alias for quest 0x0004a37b, alias quest object true
2020-08-05 09:00:48.448    trace  35524 glow quest object Ebony Blade/0x0004a38f
2020-08-05 09:00:48.448    trace  35524 Ebony Blade/0004a38f has value 4800 vs threshold 500.00: Valuable
2020-08-05 09:00:48.449    trace  35524 Valuable Item 0x0004a38f
2020-08-05 09:00:48.449    trace  35524 Player-owned true, looting belongings disallowed: Ebony Blade/0x0004a38f
2020-08-05 09:00:48.449    trace  35524 Collectible REFR 0x0004a39d overrides Legality PlayerOwned for Ebony Blade/0x0004a38f
2020-08-05 09:00:48.449    trace  35524 Trigger glow for Ebony Blade/0x0004a39d
```

**Comment by SteveTownsend** [comment]
Here is the output with modified logic to respect Quest Object instruction over Collectibility. This fix is in for v2.2.0.0.
```
2020-08-05 10:15:51.120    trace  21020 Process REFR 0x0004a39d with base object Ebony Blade/0x0004a38f
2020-08-05 10:15:51.120    trace  21020 Collectible Item 0x0004a38f
2020-08-05 10:15:51.120    trace  21020 Quest Item confirmed in alias for quest 0x0004a37b, alias quest object true
2020-08-05 10:15:51.120    trace  21020 glow quest object Ebony Blade/0x0004a38f
2020-08-05 10:15:51.120    trace  21020 Ebony Blade/0004a38f has value 4800 vs threshold 500.00: Valuable
2020-08-05 10:15:51.120    trace  21020 Valuable Item 0x0004a38f
2020-08-05 10:15:51.120    trace  21020 Player-owned true, looting belongings disallowed: Ebony Blade/0x0004a38f
2020-08-05 10:15:51.120    trace  21020 check bound object Ebony Blade/0x0004a38f with type 41
2020-08-05 10:15:51.120    trace  21020 Player location is excluded as restricted population center for this item
```
